src/H5Cdbg.c

```diff
@@ -1203,3 +1197,478 @@ H5C_verify_entry_type(H5C_t *cache_ptr, haddr_t addr, const H5C_class_t *expecte
     FUNC_LEAVE_NOAPI(ret_value)
 } /* H5C_verify_entry_type() */
 #endif /* NDEBUG */
+
+/*-------------------------------------------------------------------------
+ * Function:    H5C_def_auto_resize_rpt_fcn
+ *
+ * Purpose:     Print results of a automatic cache resize.
+ *
+ *        This function should only be used where HDprintf() behaves
+ *        well -- i.e. not on Windows.
+ *
+ * Return:      void
+ *
+ * Programmer:  John Mainzer
+ *        10/27/04
+ *
+ *-------------------------------------------------------------------------
+ */
+void
+H5C_def_auto_resize_rpt_fcn(H5C_t *cache_ptr,
+#ifndef NDEBUG
+                            int32_t version,
+#else
+                            int32_t H5_ATTR_UNUSED version,
+#endif
```

Comment:
I'm guessing you just moved this function from elsewhere, but it'd be nice to clean this up with H5_ATTR_NDEBUG_UNUSED instead if you could

Reply:
Yes - I didn't change any code for the functions themselves.

Agree about the change though, I'll put it on my TODO list and get to it in a future PR.

Reply:
Should definitely fix, but also probably better to simply cut-n-paste around and make code changes in a subsequent PR